packages/scandipwa/src/util/Form/Extract.js

```diff
@@ -273,4 +275,13 @@ export const getFieldsData = (DOM, excludeEmpty = false, ignoreTypes = [], asObj
     return output;
 };
 
+/** @namespace Util/Form/Extract/getTimeInCurrentTimezone */
+export const getTimeInCurrentTimezone = (timestamp) => {
+    const currentDate = new Date();
+    const timezone = currentDate.getTimezoneOffset() * MILLISECONDS_PER_MINUTE;
+    const timeInCurrentTimezone = new Date(timestamp).getTime() - timezone;
+
+    return getDateValue(timeInCurrentTimezone);
+};
+
```

Comment:
Btw, previously I was creating util package for date. Maybe semantically it is correct to move it there? So that all date manipulation will be in one place. 

It is located in:
```util/Manipulations/Date.js ```

What you think about it?

Reply:
Well, in general, I put the function in this class because most of the actions are with dates there, but yes, this function is not related to working with forms, and I did not notice that if there is a separate class "Date". Great idea